### PR TITLE
MOLT 1.2.0 release notes

### PR DESCRIPTION
**Pull request by taroface**
DOC-11586

### CODE REVIEW

**Review by ryanluu12345 — APPROVED** [review]

File: src/current/releases/molt.md [hunk 1]
```diff
@@ -18,6 +18,20 @@ To download the latest MOLT Fetch/Verify binary:
...
+## October 28, 2024
+
+MOLT Fetch/Verify 1.2.0 is [available](#installation).
+
+- Added `failback` mode to MOLT Fetch, which allows the user to replicate changes on CockroachDB back to the initial source database. Failback is supported for MySQL and PostgreSQL databases.
+- The [`--pprof-list-addr` flag]({% link molt/molt-fetch.md %}#global-flags), which specifies the address of the `pprof` endpoint, is now configurable. The default value is `'127.0.0.1:3031'`.
+- [Fetch modes]({% link molt/molt-fetch.md %}#fetch-mode) involving replication now state that MySQL 8.0 and later are supported for replication between MySQL and CockroachDB.
+- [Partitioned tables]({% link molt/molt-fetch.md %}#transformations) can now be moved to CockroachDB using [`IMPORT INTO`]({% link {{ site.current_cloud_version }}/import-into.md %}).
+- Improved logging for the [Fetch]({% link molt/molt-fetch.md %}) schema check phases under the `trace` logging level, which is set with [`--logging trace`]({% link molt/molt-fetch.md %}#global-flags).
+- Added a [sample Grafana dashboard](https://molt.cockroachdb.com/molt/cli/grafana_dashboard.json) for monitoring MOLT tasks.
```
Comment (florence-crl) [inline src/current/releases/molt.md:30]: this link gives a 404 error

File: src/current/releases/molt.md [hunk 2]
```diff
@@ -18,6 +18,20 @@ To download the latest MOLT Fetch/Verify binary:
...
+- The [`--pprof-list-addr` flag]({% link molt/molt-fetch.md %}#global-flags), which specifies the address of the `pprof` endpoint, is now configurable. The default value is `'127.0.0.1:3031'`.
+- [Fetch modes]({% link molt/molt-fetch.md %}#fetch-mode) involving replication now state that MySQL 8.0 and later are supported for replication between MySQL and CockroachDB.
+- [Partitioned tables]({% link molt/molt-fetch.md %}#transformations) can now be moved to CockroachDB using [`IMPORT INTO`]({% link {{ site.current_cloud_version }}/import-into.md %}).
+- Improved logging for the [Fetch]({% link molt/molt-fetch.md %}) schema check phases under the `trace` logging level, which is set with [`--logging trace`]({% link molt/molt-fetch.md %}#global-flags).
+- Added a [sample Grafana dashboard](https://molt.cockroachdb.com/molt/cli/grafana_dashboard.json) for monitoring MOLT tasks. 
+- Fetch now logs the name of the staging database used in [replication modes]({% link molt/molt-fetch.md %}#fetch-mode).
```
Comment (florence-crl) [inline src/current/releases/molt.md:31]: What is the "staging" database? There is no reference to "staging" on the `molt/molt-fetch` page.

File: src/current/releases/molt.md (hunk 1 above)
Comment (taroface) [reply src/current/releases/molt.md:30]: I believe the link will resolve once the release is actually merged. Is this correct @ryanluu12345 ?

File: src/current/releases/molt.md (hunk 2 above)
Comment (taroface) [reply src/current/releases/molt.md:31]: I think this is used in streaming. @ryanluu12345 is there a way we can clarify here? 

"staging database used for [...] in replication modes"

Comment (Jeremyyang920) [reply src/current/releases/molt.md:31]: Yes it is used for Replicator. We can maybe say staging database in the target CRDB cluster used to store replication metdata.

**Pull request merged**